lib/MusicBrainz/Server/Authentication/User.pm

```diff
@@ -33,6 +33,8 @@ sub new_from_editor
     return Class::MOP::Class->initialize($class)->rebless_instance($editor);
 }
 
+sub is_authorized { 1 }
```

Comment:
This is a workaround for when I was testing things in the browser, which I guess uses this class for cookie logins. 

This is called at https://github.com/metabrainz/musicbrainz-server/pull/240/files#diff-5f430d5eefec8efae8f240e8d2b0991fR198

In normal use of the web service `$c->user` should be lib/MusicBrainz/Server/Authentication/WS/User.pm which already has `is_authorized`.